.github/workflows/tests.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       SKIP_VM_LAUNCH: "1"
       SNAPSHOT_RESTORE: "1"
     name: System
-    runs-on: ubuntu-22.04
+    runs-on: GitHubMicrocloud
```

Comment:
Just wondering where `GitHubMicrocloud` is setup?

Reply:
Since those are paid an extra per CI minute, we had to ask IS to assign them to us. They said it was a runner group but didn't share the details on the size of said group.

Reply:
Ok thanks :)